freedomofpress/securedrop-client
# Upgrade dev dependencies

# Description

This is a draft PR until I figure out why `test_find_new_replies` and `test_find_new_messages` are failing tests (haven't looked into it yet so @gonzalo-bulnes feel free to investigate if you have time before I return to it tomorrow)

Specify the latest dev-only dependencies and update those dependencies, minus `pyqt` and `sip`

# Test Plan

0. Create a new virtual env via `make venv` and activate it
1. Run `make check`
2. - [ ] Confirm that it passes without error
3. - [ ] Confirm CI passes without error
4. - [ ] Confirm only dev depenedency files were updated (the update to `requirements.txt` is a comment-only update)
5. - [ ] Confirm you can download new files, messages, and replies (see [comment](https://github.com/freedomofpress/securedrop-client/pull/1324/files?authenticity_token=Q308Ob%2B3PPoPovE58G%2FnfRqYLy3l0Ux1Xyye96j%2BqCD5b%2Fbek9UBLF0HZ1nUB1BkiiftqNnsTGOmtFyXm%2BX4Fw%3D%3D&file-filters%5B%5D=.py#r737146693))

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:


If these changes add or remove files other than client code, the AppArmor profile may need to be updated. Please check as applicable:


If these changes modify the database schema, you should include a database migration. Please check as applicable:
